ldes-server-application/pom.xml

```diff
@@ -96,6 +96,16 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>pagination</id>
```

Comment:
kan je hier 'fragmentation-' voorzetten in lijn met de andere fragmentation profiles?